hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalNodeRpcServer.java

```diff
@@ -90,13 +91,15 @@ public class JournalNodeRpcServer implements QJournalProtocol,
         new QJournalProtocolServerSideTranslatorPB(this);
     BlockingService service = QJournalProtocolService
         .newReflectiveBlockingService(translator);
+    int handler = conf.getInt(DFS_JOURNALNODE_HANDLER_COUNT_KEY,
+        DFS_JOURNALNODE_HANDLER_COUNT_DEFAULT);
```

Comment:
Add some validation here for a valid value range, as in shouldn't be negative or so.
If it is an invalid entry, Add a warn log and use the default value.
Post that have a INFO log telling about the handler count...

Reply:
@ayushtkn Thanks for your comment, it helped me a lot. And the patch has been updated, please help me review it again. Thanks